plugins/services/src/js/data/groups/__tests__/index-test.ts

```diff
@@ -121,6 +121,144 @@ describe("Services Data Layer - Groups", () => {
         m.expect(dl.query(query, {}).pipe(take(1))).toBeObservable(expected$);
       })
     );
+
+    it(
+      "handles query for filtering enforce quota",
+      marbles(m => {
+        const marathonServiceTree = makeServiceTree(marathonGroups);
+        const roles$ = m.cold("(a|)", {
+          a: {
+            code: 200,
+            message: "OK",
```

Comment:
no action items here, just curious:
what's the significance of `message: "OK"`?